test/e2e/manifests/README.md

```diff
@@ -0,0 +1,17 @@
+# Running Assisted Installer Service in K8S
+
+Assume a k8s cluster is already setup, the following steps will bring up the necessary services for the assisted installer service,
+```
+kubectl apply -k ironic
+kubectl apply -k bmo
+kubectl apply -k namespace
+kubectl apply -k agentinstalladmission
+kubectl apply -k assisted-image-service
+kubectl apply -k assisted-service
+```
```

Comment:
What do you think about putting the URL instead of storing all the manifests in our repo?
E.g.
```
kubectl kustomize "https://github.com/openshift/assisted-service/config/default?ref=master" | kubectl apply -f -
```

Reply:
Oh, good catch, it's a mistake and I've committed unnecessary files